packages/2d/src/components/Layout.ts

```diff
@@ -729,29 +736,45 @@ export class Layout extends Node {
       this.element.style.flexGrow = '0';
       this.element.style.flexShrink = '0';
     } else {
-      this.element.style.flexGrow = this.parseValue(this.grow());
-      this.element.style.flexShrink = this.parseValue(this.shrink());
+      this.element.style.flexGrow = this.grow().toString();
+      this.element.style.flexShrink = this.shrink().toString();
     }
   }
 
   @computed()
   protected applyFont() {
-    this.element.style.fontFamily = this.parseValue(this.fontFamily());
-    this.element.style.fontSize = this.parsePixels(this.fontSize());
-    this.element.style.fontStyle = this.parseValue(this.fontStyle());
-    this.element.style.lineHeight = this.parsePixels(this.lineHeight());
-    this.element.style.fontWeight = this.parseValue(this.fontWeight());
-    this.element.style.letterSpacing = this.parsePixels(this.letterSpacing());
-
-    const wrap = this.textWrap();
-    this.element.style.whiteSpace =
-      wrap === null
-        ? ''
-        : typeof wrap === 'boolean'
-        ? wrap
-          ? 'normal'
-          : 'nowrap'
-        : wrap;
+    this.element.style.fontFamily = this.fontFamily.isInitial()
+      ? ''
+      : this.fontFamily();
+    this.element.style.fontSize = this.fontSize.isInitial()
+      ? ''
+      : `${this.fontSize()}px`;
+    this.element.style.fontStyle = this.fontStyle.isInitial()
+      ? ''
+      : this.fontStyle();
+    this.element.style.lineHeight =
+      typeof this.lineHeight() === 'string'
+        ? String(Number((this.lineHeight() as string).slice(0, -1)) / 100)
```

Comment:
The rest of the code base uses `parseFloat`, `parseInt` and `toString`, so we should try to be consistent:
```suggestion
        ? (parseFloat(this.lineHeight() as string) / 100).toString()
```